src/main/java/TaskList.java

```diff
@@ -0,0 +1,50 @@
+import java.util.ArrayList;
+
+public class TaskList {
+    public ArrayList<Task> tasks;
+
+    public TaskList() {
+        this.tasks = new ArrayList<Task>();
+    }
+
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    public void addTodo(String task) {
+        Todo newTask = new Todo(task);
+        tasks.add(newTask);
+        print("added new todo: " + task);
```

Comment:
The printing could be refactored to the ui class. Else each method is doing two unique things at once.

Reply:
Printing has been shifted to UI. TaskList methods now return a String, which will eventually be passed to MainWindow's handleUserInput()